builtin/docker/builder.go

```diff
@@ -265,6 +281,14 @@ func (b *Builder) Build(
 	return result, nil
 }
 
+func createKeyValuePairs(m map[string]string) string {
```

Comment:
```suggestion
// Translates BuildArgs from a map of key vals into a string of '--build-arg key=vals'
func createBuildArgsString(m map[string]string) string {
```

We should have a more specific name for this func since this is what it's actually doing 😄